themes/ddbasic/templates/menu/panels-pane--sub-menu--mobile.tpl.php

```diff
@@ -20,11 +20,12 @@
  * Clean out the divs and matched the design.
  */
 ?>
-<section class="navigation-wrapper <?php print $classes; ?>">
-  <div class="navigation-inner">
-    <?php if ($admin_links): ?>
-      <?php print $admin_links; ?>
-    <?php endif; ?>
-    <?php print render($content); ?>
-  </div>
-</section>
+<?php if ($is_lazy_pane_render) { ?>
+  <?php print render($content); ?>
+<?php } else { ?>
+  <section class="navigation-wrapper mobile-user-menu <?php print $classes; ?>">
+    <div class="navigation-inner">
+      <?php print render($content); ?>
+    </div>
+  </section>
+<?php } ?>
```

Comment:
This seems to remove `$admin_links` entirely from the renderend template. Is that intentional? I could not tell from the commit comment.

Reply:
I don't think they would every be set here, but re-add theme to the template to be sure.